resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json

```diff
@@ -119,7 +119,7 @@
 	"default_filament_profile": [
 		"Creality Generic PLA @Ender-3V3-all"
 	],
-	"machine_start_gcode": "M220 S100 ;Reset Feedrate\nM221 S100 ;Reset Flowrate\n\nM140 S[bed_temperature_initial_layer_single] ;Set final bed temp\nG28 ;Home\nG92 E0 ;Reset Extruder\nG1 Z2.0 F3000 ;Move Z Axis up\nM104 S[nozzle_temperature_initial_layer] ;Set final nozzle temp\nG1 X-2.0 Y20 Z0.28 F5000.0 ;Move to start position\nM190 S[bed_temperature_initial_layer_single] ;Wait for bed temp to stabilize\nM109 S[nozzle_temperature_initial_layer] ;Wait for nozzle temp to stabilize\nG1 X-2.0 Y145.0 Z0.28 F1500.0 E15 ;Draw the first line\nG1 X-1.7 Y145.0 Z0.28 F5000.0 ;Move to side a little\nG1 X-1.7 Y30 Z0.28 F1500.0 E15 ;Draw the second line\nG92 E0 ;Reset Extruder",
+	"machine_start_gcode": "M220 S100 ;Reset Feedrate\nM221 S100 ;Reset Flowrate\n\nM140 S[bed_temperature_initial_layer_single] ;Set bed temp\nG28 X Y ;Home XY axes\nM190 S[bed_temperature_initial_layer_single] ;Wait for bed temp to stabilize\nM104 S[nozzle_temperature_initial_layer] ;Set nozzle temp\nG28 Z ;Home Z axis & load bed mesh\n\nG92 E0 ;Reset Extruder\nG1 X-2.0 Y20 Z0.28 F5000.0 ;Move to start position\nM109 S[nozzle_temperature_initial_layer] ;Wait for nozzle temp to stabilize\nG1 X-2.0 Y145.0 Z0.2 F1500.0 E15 ;Draw the first line\nG1 X-1.7 Y145.0 Z0.2 F5000.0 ;Move to side a little\nG1 X-1.7 Y30 Z0.2 F1500.0 E15 ;Draw the second line\nG92 E0 ;Reset Extruder",
```

Comment:
I don't see the "load mesh"

Reply:
The G28 gcode command loads it. Creality Ender-3 V3 KE comes with a homing override config file that adds it

Reply:
To be sure, nothing needs to be changed?